### PR TITLE
🙉Monke Good

### PR DESCRIPTION
**Pull request by awtkns**
- improve test names (shown below)
- make pytest output be verbose by default
- add ability to load runners by path
- improved code loading

Screenshot 2023-11-17 at 3 23 15 PM

### CODE REVIEW

**Comment by asim-shrestha** [comment]
Maybe outside scope but should get the id in the name or folder somehow for spot testing single failures

File: bananalyzer/__main__.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        if "AgentRunner" in [getattr(base, "id", "") for base in clazz.bases]:
+            runners.append(
+                AgentRunnerClass(
+                    class_name=clazz.name,
+                    class_path=str(file_path),
+                    class_def=clazz,
+                )
+            )
+
+    return runners
```
Comment (asim-shrestha) [inline bananalyzer/__main__.py:126]: Let's update the README with this new functionality

**Comment by asim-shrestha** [comment]
What's with the deletion of the test?

**Comment by awtkns** [comment]
> Maybe outside scope but should get the id in the name or folder somehow for spot testing single failures

yes,

> What's with the deletion of the test?

its a test in the wrong directory, the null agent test file is not needed anymore with the code analyzsis

**Pull request merged**